ai/STA/Tactic/go_kick.py

```diff
@@ -192,7 +192,7 @@ def get_destination_behind_ball(self, ball_spacing, velocity=True, velocity_offs
 
         position_behind = self.game_state.ball.position - dir_ball_to_target * ball_spacing
 
-        if velocity and self.game_state.ball.velocity.norm > 20:
+        if velocity and self.game_state.ball.velocity.norm > 2000000:
```

Comment:
2 km/s c'est un peu vite?

Reply:
sky is the limit, don't speed shame the robots